images/hub/jupyterhub_config.py

```diff
@@ -92,6 +89,14 @@ def camelCaseify(s):
 
 c.KubeSpawner.namespace = os.environ.get('POD_NAMESPACE', 'default')
 
+# Max number of consecutive failures before the Hub restarts itself
+# requires jupyterhub 0.9.2
+set_config_if_not_none(
+    c.Spawner,
+    'consecutive_failure_limit',
+    'hub.consecutiveFailureLimit',
+)
```

Comment:
It took me a few attempts to see the difference: the old code set `c.JupyterHub.consecutive_failure_limit` and the new code sets `c.Spawner.consecutive_failure_limit`. Is that right?